database/layer/knex.js

```diff
@@ -0,0 +1,5 @@
+const knex = require('knex');
+const knexDbConfig = require('../knexfile');
+const usherDb = knex(knexDbConfig);
+
+module.exports = { usherDb }
```

Comment:
@k1ch  do you want to add a comment block here with some documentation on how / what the correct use is for this, maybe with an example?

Reply:
Sure I will add a jsDoc in my next PR.